<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEditorDialog2</class>
 <widget class="QWidget" name="TagEditorDialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="dm_addbut">
        <property name="text">
         <string>Add tag</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="dm_addedit"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QScrollArea" name="dm_scrollarea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>383</width>
           <height>265</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="dm_clearbut">
        <property name="text">
         <string>Remove All Tags</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dm_addedit</sender>
   <signal>returnPressed()</signal>
   <receiver>dm_addbut</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
